src/main/java/edu/hm/hafner/analysis/parser/RfLintParser.java

```diff
@@ -57,30 +129,34 @@ public Report parse(final ReaderFactory readerFactory) {
 
     @Override
     protected Optional<Issue> createIssue(final Matcher matcher, final IssueBuilder builder) {
-        String message = matcher.group(4);
-        String category = guessCategoryIfEmpty(matcher.group(1), message);
+        String message = matcher.group("message");
+        String severityStr = guessCategoryIfEmpty(matcher.group("severity"), message);
+        String ruleName = matcher.group("ruleName");
+        RfLintCategory category = RfLintCategory.CUSTOM;
+        try {
+            RfLintRuleName rule = RfLintRuleName.valueOf(ruleName);
+            category = rule.getCategory();
+        } catch (IllegalArgumentException e) {
+            // do nothing
+        }
```

Comment:
Never use exceptions for control flow, better use `EnumUtils.isValidEnum(MyEnum.class, myValue)`. Moreover, that code belongs to the enum (tell don't ask).

Reply:
Updated to use `EmailUtils.isValidEnum`

Reply:
It still should be more object oriented a part of the Enum, e.g. as method 
```
RfLintCategory.fromCategory(final String category)
```
or something similar.